recipes/configs/qwen2/7B_full.yaml

```diff
@@ -0,0 +1,78 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
```

Comment:
Could you add these to the recipe registry please?

https://github.com/pytorch/torchtune/blob/main/torchtune/_recipe_registry.py